core/server/common/src/main/java/alluxio/master/StateLockManager.java

```diff
@@ -207,10 +211,9 @@ public LockResource lockExclusive(StateLockOptions lockOptions)
           throw new TimeoutException(ExceptionMessage.STATE_LOCK_TIMED_OUT
               .getMessage(lockOptions.getGraceCycleTimeoutMs() + mForcedDurationMs));
         }
-      } catch (InterruptedException e) {
+      } finally {
```

Comment:
Would it be a problem that the success path doe another `deactivateInterruptCycle` at line 224 when the write lock is release?

Reply:
Good catch, addressed.